### PR TITLE
Filtering

### PR DESCRIPTION
**Pull request by areed08**

### CODE REVIEW

**Comment by billc2501** [comment]
Seems like it works. Looks neat! Seems ready to merge

**Review by billc2501 — APPROVED** [review]

File: src/Forum.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ function Forum() {
   async function fetchAll() {
     try {
       const response = await axios.get('https://getinvolvedapi.herokuapp.com/forum');
-      return response.data.posts_list;
+      return response.data.posts_list.reverse();
```
Comment (KennaReed) [inline src/Forum.js:21]: Thanks for reversing the forum posts. Good catch

File: src/ForumFiltering.js [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+function ForumFiltering (props){
+    var keyword1 = {name: "keyword1", word: "Sports", type: "keyWords", checked: false}
+    var keyword2 = {name: "keyword2", word: "Community", type: "keyWords", checked: false}
+    var keyword3 = {name: "keyword3", word: "School", type: "keyWords", checked: false}
+    var keyword4 = {name: "keyword4", word: "Music", type: "keyWords", checked: false}
+    var keyword5 = {name: "keyword5", word: "Art", type: "keyWords", checked: false}
+    var cost1 = {name: "cost1", word: "Free", type: "Cost", checked: false}
+    var cost2 = {name: "cost2", word: "About 5 Dollars", type: "Cost", checked: false}
+    var cost3 = {name: "cost3", word: "About 10 Dollars", type: "Cost", checked: false}
+    var cost4 = {name: "cost4", word: "More than 10 Dollars", type: "Cost", checked: false}
```
Comment (KennaReed) [inline src/ForumFiltering.js:16]: What is the "checked" attribute here?

**Review by KennaReed — APPROVED** [review]
Looks good! and it is passing with Travis, so feel free to merge once everyone else gives it the thumbs up!

File: src/ForumFiltering.js [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
-    var keyword2 = {word: "Community", type: "keyWords", checked: false}
-    var keyword3 = {word: "School", type: "keyWords", checked: false}
-    var keyword4 = {word: "Music", type: "keyWords", checked: false}
-    var keyword5 = {word: "Art", type: "keyWords", checked: false}
-    var cost1 = {word: "Free", type: "Cost", checked: false}
-    var cost2 = {word: "About 5 Dollars", type: "Cost", checked: false}
-    var cost3 = {word: "About 10 Dollars", type: "Cost", checked: false}
-    var cost4 = {word: "More than 10 Dollars", type: "Cost", checked: false}
+
+function ForumFiltering (props){
```
Comment (dathash) [inline src/ForumFiltering.js:6]: We might try and get all the filters on one line, like this:

Sports | Community | School | Music | Art
    O              O                O          O       O

If not, this works just fine.

Comment (dathash) [reply src/ForumFiltering.js:6]: Haha the formatting of the comment screwed up, but I think it's clear.

**Review by dathash — APPROVED** [review]
Looks good, no changes necessary.

**Review by jackson14furie — APPROVED** [review]
I reviewed all the changes and it looks like you did a good job with!
Go ahead and push it!

File: src/ForumFiltering.js (hunk 2 above)
Comment (areed08) [reply src/ForumFiltering.js:16]: It's so later I can go in and check all the items and if they are "checked" then they will be fetched from the backend.

**Pull request merged**